examples/ex_read_raw/main.cpp

```diff
@@ -179,7 +182,13 @@ int main(int argc, char *argv[])
 
     if (current_comp != dest_comp)
     {
-        qDebug() << "This part needs to be debugged";
+        qDebug() << "Debug Message";
```

Comment:
Remove from example. As a side note, `qFatal` will stop the application immediately.